packages/td-bank-integration/src/td-online-mart-payment-strategy.spec.ts

```diff
@@ -368,6 +376,38 @@ describe('TDOnlineMartPaymentStrategy', () => {
                 );
             });
 
+            it('call submitPayment with verification nonce when there are both physical and digital items in the cart', async () => {
```

Comment:
should we add test when do we have at least one digitalItem?

Reply:
in the test description I see `both physical and digital items in the cart` but in the reality you set `lineItems: { digitalItems: [] },`
So please improve test or the title😄

Reply:
Actually I've set `lineItems: { digitalItems: [] }` in `beforeEach` (so every test would have no digital items as default), but for this particular test I set `getCartOrThrow` to mock return `getCart()` (line 389), which contains both physical and digital items

Reply:
ok